opm/autodiff/ParallelDebugOutput.hpp

```diff
@@ -367,10 +403,10 @@ namespace Opm
                 }
 
                 // write all cell data registered in local state
-		for (const auto& pair : localState_.cellData()) {
```

Comment:
Yes that is intended as all the array are in cellData anyway. After gathering the data we extract that data from `globalCellData_` to `globalState_` using `solutionToSim`.

Also note that the globalState_ is only needed to support rudimentary matlab ouput in a parallel run. This output is (and was always) broken as the velocities will be wrong.